index.html

```diff
@@ -4987,18 +4986,18 @@ <h3>Issuer</h3>
 Relevant metadata about the <code>issuer</code> <a>property</a> is expected
 to be available to the <a>verifier</a>. For example, an <a>issuer</a> can
 publish information containing the public keys it uses to digitally sign
-<a>verifiable credentials</a> that it issued. This metadata is relevant
-when checking the proofs on the <a>verifiable credentials</a>.
+<a>verifiable credentials</a> that it issued. This metadata is relevant when
+checking the proofs on the <a>verifiable credentials</a>.
         </p>
       </section>
 
       <section class="informative">
         <h3>Issuance Date</h3>
 
         <p>
-The <code>issuanceDate</code> is expected to be within an expected range for
-the <a>verifier</a>. For example, a <a>verifier</a> can ensure that the
-issuance date of a <a>verifiable credential</a> is not in the future.
+The <code>issuanceDate</code> is expected to be within an expected range for the
+<a>verifier</a>. For example, a <a>verifier</a> can ensure the issuance date of
```

Comment:
`ensure the issuance date` -> `check that the issuance date`

Reply:
Done.